mars/deploy/oscar/tests/test_ray.py

```diff
@@ -21,7 +21,6 @@
 from mars.deploy.oscar.session import get_default_session, new_session
 from mars.deploy.oscar.tests import test_local
 from mars.serialization.ray import register_ray_serializers
-from mars.tests.conftest import *  # noqa
```

Comment:
Why remove this? We still need ray cluster fixture?

Reply:
I add the fixture to mars/conftest.py，fixture does not need to be imported explicitly any more.